sherpa/models/parameter.py

```diff
@@ -64,7 +64,7 @@
     2.0
     >>> p.val = 3
 
-Parameter limts
+Parameter limits
 ===============
```

Comment:
Sphinx won't build this if the line is too short.
```suggestion
Parameter limits
================
```